src/ycbcr.cpp

```diff
@@ -0,0 +1,33 @@
+#include "ycbcr.hpp"
```

Comment:
Will the class YCbCr be getting different functions than just getters and setters for its values? If not; I think you're better off putting those values in a struct.

Reply:
I think there is a large chance some operators will be added in the future. Also the other pixel types are classes so I think we should keep it a class for consistency.